autoload/neomake/utils.vim

```diff
@@ -379,8 +379,14 @@ function! neomake#utils#ExpandArgs(args) abort
     let isk = &iskeyword
     set iskeyword=p,h,t,r,e,%,:
     try
+        let home = expand('~')
         let ret = map(a:args,
                     \ 'substitute(v:val, '
+                    \ . '''\(\%(\\\@<!\\\)\@<!\~\)'', '
```

Comment:
I tried to include this in the pattern below, but I was having a hard time getting it to work reliably.